lib/registries/addon/overview/index/template.hbs

```diff
@@ -3,9 +3,9 @@
         {{t 'registries.overview.component_of'}}
         <OsfLink
             @route='resolve-guid'
-            @models={{array this.root.id}}
+            @models={{array this.root}}
         >
-            {{this.root.title}}
+            {{this.root}}
```

Comment:
The text should show the title of the root registration instead of the id.

Reply:
Same comment as above, this has now been updated to use the root value component name, not the guid value of the registeredFrom registration.